native/src/routes/search/components/SearchModal.js

```diff
@@ -35,14 +37,17 @@ export type PropsType = {|
   closeModal: () => void,
   navigateToLink: (url: string, language: string, shareUrl: string) => void,
   t: TFunction,
-  sendFeedback: (comment: string, query: string) => Promise<void>
+  sendFeedback: (comment: string, query: string) => Promise<void>,
+  cities: $ReadOnlyArray<CityModel>
 |}
 
-type StateType = {|
+type SearchStateType = {|
   query: string
 |}
 
-class SearchModal extends React.Component<PropsType, StateType> {
+const TranslatedFeedbackContainer = withTranslation<FeedbackContainerPropsType>('feedback')(FeedbackContainer)
```

Comment:
I don't think this is the right place for this, is it? Can't you just pass the TFunction to the component? Is it because of namespaces? Maybe using `feedback:<your-translation-key>` would work.

Reply:
I agree with you, that using everywhere the feedback:...-Prefix would be a lot more complicated. Also thought this would be the correct place because SearchModal is passing the t-prop and there seems to be responsible to set it correctly. But thinking a bit about it, the FeedbackContainer does not need a t-Function, only the Feedback.js needs it, therefore it can be moved to FeedbackContainer and the SearchModal and the FeebackModalContainer are no longer responsible for this.